src/xrCore/Threading/TaskManager.cpp

```diff
@@ -213,12 +213,63 @@ void CalcIterations()
     ttapi_dwFastIter = u32((iterations * frequency) / ((end - start) * 50000));
 }
 
+// Lainon: This implementation supports both conventional single-cpu PC configurations
+// and multi-cpu system on NUMA architecture
+size_t NUMAHardwareConcurrency() noexcept
```

Comment:
This function should be defined in `_math.h` and `_math.cpp` instead.

Reply:
fix it https://github.com/OpenXRay/xray-16/pull/1514/commits/7218c71627742bea8e033c2079492f861c26e645